python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dhcp_servers.schema.yml

```diff
@@ -149,3 +198,28 @@ keys:
                     max: 59
                     convert_types:
                     - str
+              reservations:
+                description: DHCP client reservations.
+                type: list
+                primary_key: mac_address
+                items:
+                  type: dict
+                  keys:
+                    mac_address:
+                      type: str
+                      description: Ethernet address in format - HHHH.HHHH.HHHH
+                    hostname:
+                      type: str
+                    ipv4_address:
+                      type: str
+                      description: |-
+                        Valid IPv4 address from the given subnet.
+                        This setting can only be configured when the subnet is IPv4.
```

Comment:
So this sentence seems to indicate that AVD is blocking configuring an IPv4 reservation with an IPv6 subnet - looking at the template it does not seem so.